docker/tpu/tpu_example.jsonnet

```diff
@@ -0,0 +1,33 @@
+local base = import 'templates/base.libsonnet';
```

Comment:
rather name it `tpu_test_case.jsonnet` then using missleading example

Reply:
I like `tpu_test_case.jsonnet`

right now I see `tpu_test-case.jsonnet` in your latest commits but I think 2 underscores is better than 1 underscore and 1 hyphen

Reply:
sure, let's do `tpu_test_case.jsonnet`